scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+@import "colors";
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.boxes {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  padding: 0px 0px 20px 100px;
+  margin-bottom: 50px;
+}
+
+.box {
```

Comment:
you could add display flex with direction column and align items center to center stuff